### ISSUE TITLE
DiplEd - DHARMA_DiplEdKutaraManavaLeidenOr2215

### ISSUE THREAD

**Issue by ajaniak** [erc-dharma/tfd-nusantara-philology#97]
Witnesses are declared in the encoding, is it normal? is it a mistake or do I need to make provision to add it in the code even if we are in a diplomatic edition? 
It causes issue in the display to render the Ed. 

<img width="796" alt="Capture d’écran 2022-11-22 à 11 30 07" src="https://user-images.githubusercontent.com/33328906/203291299-350b194e-baf9-4bab-aef3-6d3ee6f7f5b3.png">
<img width="351" alt="Capture d’écran 2022-11-22 à 11 30 24" src="https://user-images.githubusercontent.com/33328906/203291316-d8bf6e4e-4dbb-497d-a246-aa7e2f237797.png">
<img width="1035" alt="Capture d’écran 2022-11-22 à 11 30 34" src="https://user-images.githubusercontent.com/33328906/203291330-689aa8e4-082b-4e97-88e9-2fd6b3c913b4.png">

**Comment by arlogriffiths** [comment]
though it was intentional, I think this should be considered an encoding "mistake" in that indeed it mixes the principle of a critical edition into that of a diplomatic edition. the fact that another witness has a given reading could perhaps be represented in an xml comment or in a plain `<note>` or in a <note>` that is part of `<app>` but without using `@wit`.

@lubint : do you confirm my diagnosis?
@ajaniak : shall we ask Tim to convert the `<app>` into a `<note>`, or to use `<note>` but not `<rdg>` and `@wit` in the `<app>`, or a plain xml comment?

By the way, Tim, I think you can remove your question `<!--@tilu 2 @axja @argr: how do we indicate line numbers of the long lemma?  loc="1r4-1v1" attribute to <app> is rejected.-->`. In the case of in-line apparatus, `@loc` is not to be used at all.

**Comment by lubint** [comment]
@arlogriffiths , yes, I think this is right. I can't recall now why we (or I?) were moved to record this particular variation and not others, but changing it to an <note> or whatever other option you were suggesting (what you said after <note> is not clear to me.  I am happy to remove my question.  Perhaps I will wait until you both are done tinkering, so as not to generate a conflict of versions.

**Comment by ajaniak** [comment]
Sorry, it seems i have forgotten to answer. 
A `<note>` is fine for me. 

```
<unclear>sapadulu</unclear>ran saparane lavan ḍuṣṭa, ṣamitra lavan ḍuṣṭa<note>
    J gives <quote>samitra lavan duṣṭa, sapaduluran saparane lavan duṣṭa,</quote>
    and M <quote>samitra lavan duṣṭa, sapadulūra<subst><del rend="corrected">la</del><add place="below">n la</add></subst>van duṣṭa,</quote></note>
```

It might not be extremely precise, but the rules we have set for diplomatic edition are not unlike the critical editions. 
@lubint, is this solution, or something close to this, good enough for you?
